packages/dx-scheduler-core/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "./dist",
     "downlevelIteration": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
If you succeed in avoiding `any` in a code, please, rollback the change

Reply:
I haven't succeed in this. It involves hundreds of types, and i don't see the point in creating them for this task